.gitattributes

```diff
@@ -1 +0,0 @@
-evals/registry/data/**/*.jsonl filter=lfs diff=lfs merge=lfs -text
```

Comment:
Please simply revert the changes to this file. Do not change it and do not delete it. Thanks.

Reply:
Done now, I think